backend/app/utilities/ses/ses_init.py

```diff
@@ -0,0 +1,82 @@
+import json
+import os
+from typing import Dict
+
+import boto3
+from botocore.exceptions import ClientError
+
+TEMPLATES_FILE = "app/utilities/ses/ses_templates.json"
+TEMPLATES_DIR = "app/utilities/ses/template_files"
+
+ses_client = boto3.client(
+    "ses",
+    region_name=os.getenv("AWS_REGION"),
+    aws_access_key_id=os.getenv("AWS_ACCESS_KEY"),
+    aws_secret_access_key=os.getenv("AWS_SECRET_KEY"),
+)
+
+
+def load_templates_metadata(file_path: str) -> Dict:
+    try:
+        with open(file_path, "r") as file:
+            return json.load(file)
+    except FileNotFoundError:
+        print(f"Error: {file_path} not found.")
+        return []
+    except json.JSONDecodeError as e:
+        print(f"Error parsing {file_path}: {e}")
+        return []
+
+
+def load_file_content(file_path: str) -> str:
+    """Reads the content of a file."""
+    try:
+        with open(file_path, "r") as file:
+            return file.read()
+    except FileNotFoundError:
+        print(f"Error: File '{file_path}' not found.")
+        return ""
+
+
+templates_metadata = load_templates_metadata(TEMPLATES_FILE)
+
+
+# Function to create SES template
+def create_ses_template(template_metadata):
+    name = template_metadata["TemplateName"]
+    try:
+        text_part = load_file_content(template_metadata["TextPart"])
+        html_part = load_file_content(template_metadata["HtmlPart"])
+        if not text_part or not html_part:
+            print(f"Skipping template '{name}' missing content.")
+            return
+
+        template = {
+            "TemplateName": template_metadata["TemplateName"],
+            "SubjectPart": template_metadata["SubjectPart"],
+            "TextPart": text_part,
```

Comment:
Note: needed when user can't see HTML content, but HTML is default. https://docs.aws.amazon.com/ses/latest/APIReference/API_Template.html